cli.mjs

```diff
@@ -8,6 +8,7 @@ import { PACKAGES_ROOT, errorHandler, getJSON } from './scripts/helpers/esm.mjs'
 
 const argvNoBin = hideBin(process.argv);
 const argv = yargs(argvNoBin)
+  .scriptName("refinitiv-ui")
```

Comment:
I don't think there is any command with such name.

Reply:
This is optional I add it to prevent confusion because we have the two cli.mjs files

Reply:
![image](https://user-images.githubusercontent.com/117723407/228219837-cccb3460-4d86-4743-9fc6-cfc5c03a7585.png)

let's remove this `scriptName()` as it only creates confusion to anyone using `--help` option. 
While `node refinitiv-ui test core` will never work, `node cli.mjs test core` will do the job.

Reply:
That's right. removed